hollow-explorer-ui/src/main/java/com/netflix/hollow/explorer/ui/HollowExplorerUI.java

```diff
@@ -105,13 +114,16 @@ public HollowReadStateEngine getStateEngine() {
             return client.getStateEngine();
         return stateEngine;
     }
-    
-    public void setHeaderDisplayString(String str) {
-        this.headerDisplayString = str;
+
+    public void addToHeaderStringMap(String var, String str) {
+        if (headerDisplayMap == null) {
+            headerDisplayMap = new HashMap<String, String>(5); //1.5 times the initial count of elements(2)
```

Comment:
Sorta feels like this is an unnecessary optimization - should we just initialize headerDisplayMap as an empty HashMap when we declare it?
Also, any need to have an addTo method or should we have users call getHeaderDisplayMap and then `.put`? Or it the getter only meant for the UI? If so, the getter should probably return an immutable view of the map.

Reply:
Will remove the size of hashmap.
This method gets called from ExploreCommand in cinder-cli. Added that method name for more readability. Correct, the getter method for this map is being called in HollowExplorerPage (UI).